frontend/packages/dev-console/src/components/helm/HelmResourcesListComponent.tsx

```diff
@@ -11,6 +11,7 @@ const HelmResourcesListComponent: React.FC<TableProps> = (props) => {
     <Table
       {...props}
       aria-label="Resources"
+      defaultSortField={'kind'}
```

Comment:
use a variable instead and initialize it to 'kind' outside the return statement.

Reply:
This is sufficient really:
```suggestion
      defaultSortField="kind"
```

This functional component could be simplified to omit the return statement altogether and inline return the `<Table`